chp08_objects/example_08_02_twoobjects/sketch.js

```diff
@@ -21,25 +21,27 @@ function draw() {
   car2.display();
 }
 
-// Even though there are multiple objects, we still only need one constructor function. 
-// No matter how many cookies we make, only one cookie cutter is needed.Isn’t object-oriented programming swell?
-function Car(tempC, tempXpos, tempYpos, tempXspeed) { // The Constructor is defined with arguments.
-  this.c = tempC;
-  this.xpos = tempXpos;
-  this.ypos = tempYpos;
-  this.xspeed = tempXspeed;
+// Even though there are multiple objects, we still only need one constructor function.
+// No matter how many cookies we make, only one cookie cutter is needed. Isn’t object-oriented programming swell?
+class Car {
+    constructor(tempC, tempXpos, tempYpos, tempXspeed) { // The Constructor is defined with arguments.
+        this.c = tempC;
+        this.xpos = tempXpos;
+        this.ypos = tempYpos;
+        this.xspeed = tempXspeed;
+    }
 
-  this.display = function() {
-    stroke(0);
-    fill(this.c);
-    rectMode(CENTER);
-    rect(this.xpos,this.ypos,20,10);
-  }
+    display() {
+        stroke(0);
```

Comment:
do these indents match the rest of the code? (two spaces?)

Reply:
Ah! No, my mistake. My editor is set to 4 and I must have missed this. I'll correct this.